src/js/core/Core.js

```diff
@@ -657,9 +657,9 @@ export const onCall = async (message: CoreMessage) => {
 
         if (response) {
             if (method.name === 'rebootToBootloader' && response.success) {
-                // trezord may not detect auto reboot
-                // wait for device to switch to bootloader
-                await resolveAfter(501);
+                // Wait for device to switch to bootloader
+                // This delay is crucial see trezor-firmware#1983
```

Comment:
Let's use the full link

```suggestion
                // This delay is crucial see https://github.com/trezor/trezor-firmware/issues/1983
```